hassio/src/dialogs/system_checks/dialog-hassio-system-checks.ts

```diff
@@ -0,0 +1,117 @@
+import {
+  CSSResult,
+  customElement,
+  html,
+  internalProperty,
+  LitElement,
+  property,
+  TemplateResult,
+} from "lit-element";
+import { fireEvent } from "../../../../src/common/dom/fire_event";
+import "../../../../src/components/ha-circular-progress";
+import { createCloseHeading } from "../../../../src/components/ha-dialog";
+import "../../../../src/components/ha-settings-row";
+import "../../../../src/components/ha-svg-icon";
+import { extractApiErrorMessage } from "../../../../src/data/hassio/common";
+import { setCheckOption } from "../../../../src/data/hassio/resolution";
+import { Supervisor } from "../../../../src/data/supervisor/supervisor";
+import { showAlertDialog } from "../../../../src/dialogs/generic/show-dialog-box";
+import { haStyle, haStyleDialog } from "../../../../src/resources/styles";
+import type { HomeAssistant } from "../../../../src/types";
+import { SystemChecksParams } from "./show-dialog-system-checks";
+
+@customElement("dialog-hassio-system-checks")
+class HassioSystemChecksDialog extends LitElement {
+  @property({ attribute: false }) public hass!: HomeAssistant;
+
+  @property({ attribute: false }) public supervisor!: Supervisor;
+
+  @internalProperty() private _opened = false;
+
+  protected render(): TemplateResult {
+    return html`
```

Comment:
```suggestion
    return html`
    
    if (!this.supervisor) {
      return html``
    }
```
Also `this._opened` would not be needed anymore